reascripts/ReaSpeech/source/ASRActions.lua

```diff
@@ -0,0 +1,176 @@
+--[[
+
+  ASRActions.lua - Actions for ASR plugin
+
+]]--
+
+ASRActions = Polo {
+  new = function(plugin)
+    return {
+      plugin = plugin
+    }
+  end,
+}
```

Comment:
i could imagine formalizing this implicit notion that ancillary plugin modules (actions and controls) are initialized with and contain a reference to the owning plugin.